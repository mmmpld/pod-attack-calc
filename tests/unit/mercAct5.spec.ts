import { describe, it } from 'vitest'
import breakpoints from './breakpoints.js'
const characterId = 9

describe('Merc Act5 default values', () => {
  it('bash attack', async () => {
    await breakpoints('15', '1.47', { charactersSelected: characterId })
  })
})
