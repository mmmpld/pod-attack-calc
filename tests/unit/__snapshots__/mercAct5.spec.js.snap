// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Merc Act5 default values > bash attack 1`] = `
Wrapper {
  "selector": ".breakpoint-table tbody",
}
`;

exports[`Merc Act5 default values bash attack 1`] = `
<tbody>
  <tr class="highlight-current-row">
    <th>0</th>
    <td>15</td>
    <td>1.47</td>
  </tr>
  <tr class="">
    <th>9</th>
    <td>14</td>
    <td>1.56</td>
  </tr>
  <tr class="">
    <th>18</th>
    <td>13</td>
    <td>1.66</td>
  </tr>
  <tr class="">
    <th>30</th>
    <td>12</td>
    <td>1.78</td>
  </tr>
  <tr class="">
    <th>48</th>
    <td>11</td>
    <td>1.92</td>
  </tr>
  <tr class="">
    <th>75</th>
    <td>10</td>
    <td>2.08</td>
  </tr>
  <tr class="">
    <th>125</th>
    <td>9</td>
    <td>2.27</td>
  </tr>
</tbody>
`;
